python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -18,13 +25,15 @@
 mypath = os.path.dirname(os.path.abspath(__file__))+'/'
 
 
-def CoordinateTransform(xyin, za, mode, inr=0., cent=np.array([[0.], [0.]])):
+def CoordinateTransform(xyin, za, mode, inr=0., pa=0.,
+                        cent=np.array([[0.], [0.]]),
+                        time='2020-01-01 10:00:00'):
     """Transform Coordinates given za and inr. Inputs are x,y point list,
     zenith angle, mode, rotator angle, centerposition
 
     Parameters
     ----------
-    xyin : `np.ndarray`, (N, 2)
+    xyin : `np.ndarray`, (2, N)
         Input coordinates.
         Unit is degree for sky, mm for PFI, and pixel for MCS.
```

Comment:
Tiny suggestion for absolute clarity: "Unit is (Ra, Dec) in degrees for sky" ...?